catalyst/core/runner.py

```diff
@@ -319,8 +319,8 @@ def _run_epoch(self, stage: str, epoch: int):
             self.state.loader_len = len(loader)
             self.state.need_backward = loader_name.startswith("train")
             if isinstance(self.model, dict):
-                for submodel in self.model:
-                    self.model[submodel].train(self.state.need_backward)
+                for model in self.model.items():
```

Comment:
No, `items()` returns a tuple `(key, value)`

You need `values()`

Reply:
Thank you, fixed it